driver-mapping/src/test/java/com/datastax/driver/mapping/MapperTest.java

```diff
@@ -416,7 +417,7 @@ public Thread apply(Void v) {
         executeFunctionAndTestForException(u, mapper, new Function<Void, Thread>() {
             @Override
             public Thread apply(Void v) {
-                mapper.save(u, null);
```

Comment:
The idea was to test every entry point where the thread check was added. In this case that would be save method with and without options. I might be missing something but this would be the exact same as the above, and would miss save method where options are passed in wouldn't it?

If so applies to the methods below as well.